modal/experimental.py

```diff
@@ -155,6 +155,16 @@ def _networked(func):
     @wraps(func)
     def wrapper(*args, **kwargs):
         import os
+        import socket
+
+        hostname = socket.gethostname()
+        addr_info = socket.getaddrinfo("i6pn.modal.local", None, socket.AF_INET6)[0][4][0]
```

Comment:
```suggestion
        addr_info = socket.getaddrinfo("i6pn.modal.local", None, socket.AF_INET6)[0][4][0]
```

nit: would be more readable to unpack or comment the `[0][4][0]` indexing, even if it costs a couple extra LoC.

Reply:
the [0][4][0] is just a magic incantation that you need to know, I don't think there's anything smart about it 🤷‍♂️

maybe we can make it a helper function to get the i6pn